audiotoolbox.sh

```diff
@@ -37,4 +37,20 @@ audioclip(){
 	rm "temp_$1"
 }
 
-
+flacer(){
+	echo Do you want a funny version? [y/N]
```

Comment:
Not informative. Gets in the way of user comfort, is asked every time. If you want to add such configuration option I'd recommend implementing a config file.

Reply:
That's true, will do.